2554: Clean up mui

f1sh1918 opened the pull request:
### Short Description

<!-- Describe this PR in one or two sentences. -->

### Proposed Changes

<!-- Describe this PR in more detail. -->

- migrate ActivityLogController.tsx and fix 
- remove blueprint mocks from tests
- remove blueprint from CreateUserDialog.tsx
- migrate VerificationListItem.tsx
- migrate JsonFieldElemental.tsx
- migrate ResetPasswordController.tsx

### Side Effects

<!-- List all related components that have not been explicitly changed but may be affected by this PR -->

- should be none

### Testing

- check create user dialog (Benutzer verwalten)
- check ApplicationDetails for Attachment Download Button and done
- go to "Passwort vergessen" on login screen, click the pw reset link and change password

### Resolved Issues

<!-- List all issues which should be closed when this PR is merged. -->

Fixes: #2554

bahaaTuffaha commented on administration/src/bp-modules/auth/ResetPasswordController.tsx:
```diff
@@ -73,35 +72,59 @@ const ResetPasswordController = (): ReactElement => {
+                fullWidth
+                autoFocus={false}
+              />
             </FormControl>
```
I think `Textfield` got FormControl inherited: https://mui.com/material-ui/api/text-field/#inheritance
 
```suggestion
<PasswordInput
                label={t('newPassword')}
                placeholder='Passwort'
                setValue={setNewPassword}
                value={newPassword}
                disabled={loading}
                fullWidth
                autoFocus={false}
              />
```

and at `PasswordInput`:

```js
  return (
    <TextField
      placeholder={p.placeholder}
      type={passwordHidden ? 'password' : 'text'}
      size='small'
      required
      label={p.label}
      autoComplete='on'
      autoFocus={p.autoFocus}
      fullWidth={p.fullWidth}
      disabled={p.disabled}
      value={p.value}
      sx={p.sx}
      slotProps={{
        input: {
          endAdornment: (
            <InputAdornment position='end'>
              <Tooltip title={t(passwordHidden ? 'showPassword' : 'hidePassword')}>
                <IconButton
                  aria-label={t(passwordHidden ? 'showPassword' : 'hidePassword')}
                  onClick={() => {
                    setPasswordHidden(!passwordHidden)
                  }}
                  edge='end'>
                  {passwordHidden ? <Visibility /> : <VisibilityOff />}
                </IconButton>
              </Tooltip>
            </InputAdornment>
          ),
        },
      }}
      onChange={event => p.setValue?.(event.currentTarget.value)}
    />
  )
```

f1sh1918 commented on administration/src/bp-modules/users/CreateUserDialog.tsx:
```diff
@@ -99,22 +98,21 @@ const CreateUserDialog = ({
+        <BaseCheckbox
+          label={
+            <>
+              {' '}
```
auto generated by formatting

f1sh1918 commented:
> Tested on chrome ✅️
> 
> I noticed that there is some `Table2` from mui that needs replacement I think. And left some suggestions...

I addressed all issues!

kalairubachandrakumaran-oss commented:
Test result:
- check create user dialog (Benutzer verwalten) ✅ ℹ️ related small issue: #2711 #2712
- check ApplicationDetails for Attachment Download Button and done ✅ 
- go to "Passwort vergessen" on login screen, click the pw reset link and change password ✅
